keyboards/crkbd/keymaps/nimishgautam/keymap.c

```diff
@@ -0,0 +1,591 @@
+
```

Comment:
Please add a GPLv2 compatible copyright notice to this file.

Reply:
added. Thanks again for the review